freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/cleanup/CleanupService.java

```diff
@@ -145,6 +159,26 @@ public Pair<Set<String>, Map<String, String>> revokeCerts(Long stackId, Set<Stri
         return Pair.of(certCleanupSuccess, certCleanupFailed);
     }
 
+    public Optional<String> cleanUpStructuredEvents(Long stackId) throws TransactionExecutionException {
```

Comment:
please create a separate service for this. This class is used for cleaning up stuff which is present in FreeIPA